ACP2E-228: [Documentation] Add notice that the cookie path can't contain any cookie parameters except path to folders (#1535)
magento/merchdocs: src/stores/compliance-cookie-restriction-mode.md

    - Enter the **Cookie Lifetime** in seconds.

>     - If you want to make cookies available to other folders, enter the **Cookie Path**. To make the cookies available anywhere in the site, enter a forward slash (`/`). It can't contain any other cookie parameters, except the cookie path.

Reviewer: ```suggestion
    - If you want to make cookies available to other folders, enter the **Cookie Path**. To make the cookies available anywhere in the site, enter a forward slash (`/`). This value can contain only the cookie path, and **_cannot_** contain any other cookie parameters.
```

Author: Requested changes were applied successfully

>     - To make the cookies available to a subdomain, enter the subdomain name in the **Cookie Domain** field (`subdomain.yourdomain.com`). To make cookies available to all subdomains, enter the domain name preceded by a period (`.yourdomain.com`). It can't contain any other cookie parameters, except the cookie domain.

Reviewer: ```suggestion
    - To make the cookies available to a subdomain, enter the subdomain name in the **Cookie Domain** field (`subdomain.yourdomain.com`). To make cookies available to all subdomains, enter the domain name preceded by a period (`.yourdomain.com`). This value can contain only the cookie domain, and **_cannot_** contain any other cookie parameters.
```

Author: Proposed changes were applied successfully
